src/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Message } from '../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { safeLoad } from "js-yaml";
+import { AutoRestPluginHost } from "../../jsonrpc/plugin-host";
+import { run } from "../../azure-openapi-validator";
+import { AssertValidationRuleCount, CollectTestMessagesFromValidator, ReadFileAsString } from './utilities/tests-helper';
+
+
+@suite class CompositeAzureTest {
```

Comment:
What's the reason you're calling this class CompositAzureTest? I'm just curious, on why this name was chosen?